src/components/AccountSettings/AdminPanel.tsx

```diff
@@ -154,7 +114,7 @@ class AdminPanel extends Component<Props, State> {
                   type="submit"
                   className="btn btn-outline-primary"
                   onClick={(): void => {
-                    const { alert } = this.props;
+                    const { alert } = props;
```

Comment:
I think `const { alert } = props;` can be lifted up to outside the `workerPanel` definition so it doesn't need to be repeated within each `onClick`